nomad/structs/structs.go

```diff
@@ -1146,7 +1146,7 @@ func (sc *ServiceCheck) Hash(serviceId string) string {
 }
 
 const (
-	NomadConsulPrefix = "nomad"
+	NomadConsulPrefix = "1729nomad"
```

Comment:
Yeah I don't care much either. Fixing a test, might as well make it what Alex suggested so that it's more human readable.